File: app/src/at-dinhtruong/java/asiantech/internship/summer/models/Image.java
package asiantech.internship.summer.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Image {
    @SerializedName("image_id")
    @Expose
    private String imageId;
    @SerializedName("url")
    @Expose
    private String url;
    @SerializedName("type")
    @Expose
    private String type;

    public String getImageId() {
        return imageId;
    }

    public String getUrl() {
        return url;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

}

File: app/src/at-dinhtruong/java/asiantech/internship/summer/kotlin/model/Image.kt
package asiantech.internship.summer.kotlin.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Image {

    @SerializedName("image_id")
    @Expose
    var imageId: String? = ""

    @SerializedName("permalink_url")
    @Expose
    var permalinkUrl: String? = ""

    @SerializedName("url")
    @Expose
    var url: String? = ""

    @SerializedName("type")
    @Expose
    var type: String? = ""

    @SerializedName("thumb_url")
    @Expose
    var thumbUrl: String? = ""

    @SerializedName("created_at")
    @Expose
    var createdAt: String? = ""
}
